build/proj/CoreBuild.proj

```diff
@@ -29,15 +29,38 @@
     <CommonBuildProperty Include="CIBuild=$(CIBuild)" />
     <CommonBuildProperty Include="RepoRoot=$(RepoRoot)" />
     <CommonBuildProperty Include="VersionsPropsPath=$(RepoRoot)build\import\Versions.props" />
+    <CommonBuildProperty Include="ProductProjects=$(ProductProjects)" />
+    <CommonBuildProperty Include="PackageProjects=$(PackageProjects)" />
+    <CommonBuildProperty Include="TestProjects=$(TestProjects)" />
   </ItemGroup>
 
   <ItemGroup>
     <SolutionRestoreAndBuildProperty Include="@(CommonBuildProperty)"/>
     <SolutionRestoreAndBuildProperty Include="__DeployProjectOutput=$(Deploy)" Condition="'$(Deploy)' != ''" />
   </ItemGroup>
 
-  <Target Name="LogInitialMessage">
-    <Message Text="Building $(SolutionFileName) [$(Configuration)]...%0a" Importance="high" />
```

Comment:
I removed this target as it only logged a message. I added a message at the top of each target in this file instead.